torch/utils/hipify/hipify_python.py

```diff
@@ -446,6 +448,7 @@ def hip_header_magic(input_string):
         return output_string
 
     # Rough logic to detect if we're inside device code
+    hasDeviceLogic: int
     hasDeviceLogic = "hipLaunchKernelGGL" in output_string
```

Comment:
I'd expect this to need `int()` around it to not make mypy unhappy - it's a `bool` that gets turned into `int` on the next line otherwise.

Reply:
mypy was smart enough to convert the `bool` to an integer